pkg/policy/repository.go

```diff
@@ -762,7 +762,7 @@ func (p *Repository) computePolicyEnforcementAndRules(securityIdentity *identity
 		if !egress || !hasEgressDefaultDeny { // short-circuit len()
 			if len(r.Egress) > 0 || len(r.EgressDeny) > 0 {
 				egress = true
-				if *r.EnableDefaultDeny.Egress {
+				if r.EnableDefaultDeny.Egress == nil || *r.EnableDefaultDeny.Egress {
```

Comment:
I'd suggest sanitizing the rules properly instead, then we shouldn't need extra nil checks much later in processing like this.
```suggestion
				if *r.EnableDefaultDeny.Egress {
```

Reply:
done.